pcsx2/gui/GlobalCommands.cpp

```diff
@@ -444,10 +444,14 @@ namespace Implementations
 		ScopedCoreThreadPause paused_core;
 		paused_core.AllowResume();
 
-		g_Pcsx2Recording ^= 1;
+		if (wxGetApp().HasGUI())
+		{
+			sMainFrame.VideoCaptureToggle();
+			return;
+		}
 
 		GetMTGS().WaitGS(); // make sure GS is in sync with the audio stream when we start.
-		if (g_Pcsx2Recording)
+		if (g_Pcsx2Recording ^= 1)
```

Comment:
can we change this to `g_Pcsx2Recording = !g_Pcsx2Recording` at the least to get rid of the bitwise fanciness.  Also perhaps it might be a good idea to keep it outside of the `if`, not sure how the rest of the pcsx2 devs feel about it, but assignment in the if clause for me isn't the prettiest thing

Reply:
Sure, since it won't really change performance. Was just a stylistic choice here.